unbxdAutosuggest.js

```diff
@@ -1486,6 +1566,9 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 				//main zero side not zero
 				if ((mainlen == 0) && (sidelen != 0)) {
 					this.options.sideTpl.forEach(function (key) {
+						if (self.options.sortByLength && (key == 'topQueries' || key == 'keywordSuggestions')) {
```

Comment:
Actually, we can use `key !== "SORTED_SUGGESTIONS"`. Overall bundling code size will reduce.

Reply:
How does logically you explain `key !== "SORTED_SUGGESTIONS"` ? What if tomorrow, we have more suggestion, let's consider the GroupOfSuggestion? I mean, I can understand this `(self.options.sortByLength && (key == 'topQueries' || key == 'keywordSuggestions'))`

Reply:
I mean, I dont see any problem as such that

Reply:
@ganeshparsads Using !sorted suggestions will include infields too